src/DynamoCoreWpf/Views/CodeCompletion/CodeCompletionEditor.xaml.cs

```diff
@@ -186,7 +188,15 @@ private void OnTextChanged(object sender, EventArgs e)
             if (WatermarkLabel.Visibility == Visibility.Visible)
             {
                 WatermarkLabel.Visibility = Visibility.Collapsed;
-                InnerTextEditor.ShowLineNumbers = true;
+                this.InnerTextEditor.ShowLineNumbers = dynamoViewModel.ShowCodeBlockLineNumber;
```

Comment:
Why does this line need to be here and not when the UI component has been initialized?

Reply:
Hi @aparajit-pratap, we discussed about this already when we first time merging the PR from contractor, but you may not be aware. If you put this in the constructor, the code block node will not be displayed correctly, meaning that line number will be there from the begining, and when you try to type code in it, the code overlaps with the line number. So this would be the best place to put so that line number are not added for empty CBN